packages/web3-eth-accounts/src/account.ts

```diff
@@ -67,6 +68,8 @@ import type {
 	SignResult,
 } from './types';
 
+const secp256k1 = ethereumCryptography.secp256k1 ?? ethereumCryptography;
+
```

Comment:
What was the reason for needing this?

Reply:
in case of ESM `secp256k1` was undefined